libnetwork/test/integration/dnet/helpers.bash

```diff
@@ -149,21 +132,13 @@ function start_dnet() {
     # Try discovery URLs with or without path
     neigh_ip=""
     neighbors=""
-    if [ "$store" = "zookeeper" ]; then
-	read discovery provider address < <(parse_discovery_str zk://${bridge_ip}:2182)
-    elif [ "$store" = "etcd" ]; then
-	read discovery provider address < <(parse_discovery_str etcd://${bridge_ip}:42000/custom_prefix)
-    elif [ "$store" = "consul" ]; then
-	read discovery provider address < <(parse_discovery_str consul://${bridge_ip}:8500/custom_prefix)
-    else
 	if [ "$nip" != "" ]; then
 	    neighbors=${nip}
 	fi
 
 	discovery=""
 	provider=""
 	address=""
```

Comment:
This indentation is technically "wrong" now, but IMO it's not important enough to put this through CI/edits again (maybe something someone can fix as a separate PR).

Reply:
oh, yeah, quite likely; I mostly tried to make my "remote X" commits clear on what I was removing, which I why I removed the bits that were related, but I think none of these are used anymore, so we likely can just remove everything here. now (I'll keep that for a follow-up)